flask_static_digest/__init__.py

```diff
@@ -22,8 +23,23 @@ def init_app(self, app):
         :return: None
         """
         app.config.setdefault("FLASK_STATIC_DIGEST_BLACKLIST_FILTER", [])
-        app.config.setdefault("FLASK_STATIC_DIGEST_GZIP_FILES", True)
         app.config.setdefault("FLASK_STATIC_DIGEST_HOST_URL", None)
+        app.config.setdefault("FLASK_STATIC_DIGEST_GZIP_FILES", None)
+        app.config.setdefault("FLASK_STATIC_DIGEST_COMPRESSION", ["gzip"])
+
+        compression = app.config["FLASK_STATIC_DIGEST_COMPRESSION"] = set(
```

Comment:
Can you remove this?

Reply:
I prefer one shorter variable instead of long string keys in dicts, but it is an extra layer of indirection.

Reply:
It's mainly for code clarity.

At a glance when you call `compression.add` or `compression.discard` it looks like you're modifying a local variable that never leaves that function, but the expected outcome we want is the `app.config["FLASK_STATIC_DIGEST_COMPRESSION"]` dictionary to be mutated by the list operations.

In my opinion using `a = b = expression` is harder to see that b will be mutated.